i18n/ui/lbm-ui_en-US.po

```diff
@@ -346,6 +346,10 @@ msgstr "We're all out of coins!"
 msgid "Please try again later."
 msgstr "Please try again later."
 
+#: On screen: limit-reached
+msgid "<div> OK </div>"
+msgstr "<div> OK </div>"
+
```

Comment:
Hehe, you just found a bug on the limit-reached screen. i18n tag should be on the inner div.